AUTHORS

```diff
@@ -10,6 +10,7 @@ Borg authors ("The Borg Collective")
 - Marian Beermann <public@enkore.de>
 - Daniel Reichelt <hacking@nachtgeist.net>
 - Lauri Niskanen <ape@ape3000.com>
+- Abdel-Rahman A. (Abogical)
```

Comment:
1. I may chnage my email in the future
2. my family name is my first name, making it an ugly name, so I choose to hide it.